tools/apigee-sackmesser/cmd/report/report.sh

```diff
@@ -81,13 +88,15 @@ fi
 loginfo "Running Apigeelint on Proxies"
 mkdir -p "$export_folder/apigeelint/proxies"
 
-while IFS= read -r -d '' proxyexportpath
-do
-    proxyname=$(basename "$proxyexportpath")
-    logdebug "Running Apigeelint on: $proxyexportpath"
-    apigeelint -s "$proxyexportpath/apiproxy" -f html.js > "$export_folder/apigeelint/proxies/$proxyname.html" || true # apigeelint exits on error but we want to continue
-    apigeelint -s "$proxyexportpath/apiproxy" -f json.js > "$export_folder/apigeelint/proxies/$proxyname.json" || true #
-done <   <(find "$export_folder/$organization/proxies" -type d -mindepth 1 -maxdepth 1 -print0)
+if [ "$skip_api_lint" == "false" ]; then
```

Comment:
For consistency can we please add the same skipping because of env var loginfo as for the skip_api_export?

Reply:
@danistrebel  Done!!

Reply:
@danistrebel  Done